x-pack/plugins/security_solution/public/management/pages/event_filter/view/components/form/index.test.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EventFilterForm } from '.';
+import { RenderResult, act, render } from '@testing-library/react';
+import { fireEvent } from '@testing-library/dom';
+import { stubIndexPatternWithFields } from 'src/plugins/data/common/index_patterns/index_pattern.stub';
+import { getInitialExceptionFromEvent } from '../../../store/utils';
+import { Provider } from 'react-redux';
+import { useFetchIndex } from '../../../../../../common/containers/source';
+import { ThemeProvider } from 'styled-components';
+import { createGlobalNoMiddlewareStore, event } from '../../../test_utils';
+import { getMockTheme } from '../../../../../../common/lib/kibana/kibana_react.mock';
+import { NAME_ERROR, NAME_PLACEHOLDER } from './translations';
+import { useCurrentUser, useKibana } from '../../../../../../common/lib/kibana';
+
+jest.mock('../../../../../../common/lib/kibana');
+jest.mock('../../../../../../common/containers/source');
+
+const mockTheme = getMockTheme({
+  eui: {
+    paddingSizes: { m: '2' },
+  },
+});
+
+describe('Event filter form', () => {
+  let component: RenderResult;
+  let store: ReturnType<typeof createGlobalNoMiddlewareStore>;
+
+  const renderForm = () => {
+    const Wrapper: React.FC = ({ children }) => (
```

Comment:
FYI - there is a testing utility named `createAppRootMockRenderer()` here `x-pack/plugins/security_solution/public/common/mock/endpoint/app_context_render.tsx` that will return you back an entire setup that should avoid you having to create the wrapper as well as the store setup adn mocked global libraries from security soluton.